### PR TITLE
Beautiful landing page

### PR DESCRIPTION
**Pull request by tischi**

### CODE REVIEW

**Comment by tischi** [comment]
@manerotoni OK, If I merge this?

File: Gemfile [hunk 1]
```diff
@@ -8,3 +8,5 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 ruby '>=2.7.1'
 
 gem 'github-pages', group: :jekyll_plugins
+
+gem "webrick", "~> 1.7"
```
Comment (tischi) [inline Gemfile:12]: @unode Do you have any idea whether this is needed? I did not put this dependency there, it somehow appeared on its own. No idea how...

Comment (unode) [reply Gemfile:12]: It's unrelated to bootstrap so likely not needed. Probably got added while you were working on the pages on your computer.

**Comment by tischi** [comment]
@manerotoni I'll just merge it. I don't think this is doing any harm.

**Pull request merged**